docs/usage/validation_errors.md

```diff
@@ -1165,29 +1180,18 @@ class Model(BaseModel):
     x: str
 
 
+Model.model_validate(Model(x='test'), strict=True)  # OK
+
 try:
     Model.model_validate({'x': 'test'}, strict=True)
-except ValidationError as exc_info:
-    assert exc_info.errors()[0]['type'] == 'model_class_type'
-```
-
-Valid example:
-
-```py
-from pydantic import BaseModel
-
-
-class Model(BaseModel):
-    x: str
-
-
-Model.model_validate(Model(x='test'), strict=True)
+except ValidationError as exc:
+    print(repr(exc.errors()[0]['type']))
```

Comment:
I think you forgot to add `#> model_class_type ` as `print` statement output.
I am curious how the test passes without this.

Reply:
it means it's not hitting the error, which is why I changed to `print` from the way it was before — the old way, since it did an assert, you couldn't tell if it was hitting the error. Good catch, I'll fix this.